Pipfile

```diff
@@ -21,4 +21,4 @@ httpie = "*"
 "psycopg2-binary" = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.7"
```

Comment:
I would ask Steven before changing the Python version. It potentially could mess things up, but is probably fine. If you're working on form validation, you shouldn't make any changes like this to keep things organized.

If you're modifying a file like this, it should be in a separate commit/PR.

You can go ahead and change this back to "3.6"